UltraStar Play/Assets/Common/Model/Setting/GraphicSettings.cs

```diff
@@ -14,6 +14,20 @@ public class GraphicSettings
     public bool useImageAsCursor = true;
     public bool showLyricsOnNotes;
     public bool analyzeBeatsWithoutTargetNote = true;
-    public string themeName = "BaseTheme";
+    public string themeName = ThemeManager.DEFAULT_THEME;
     public bool AnimateSceneChange { get; set; } = true;
+
+    public string CurrentThemeName
```

Comment:
- This logic should be in the ThemeManager. The Settings classes are only to store values, like Dto classes.